### PR TITLE
Addition of Pre-Made Pizzas

### PR DESCRIPTION
**Pull request by rwideman94**

### CODE REVIEW

File: PizzaData/Models/Orders.cs [hunk 1]
```diff
@@ -8,6 +8,7 @@ public class Orders
     {
         public List<Pizzas> Pizzas { get; set; }
         public List<int> SidesIds { get; set; }
+        public List<int> PreMadePizazIds { get; set; }
```
Comment (steamwings) [inline PizzaData/Models/Orders.cs]: Please fix

File: PizzaShop/Controllers/OrdersController.cs [hunk 2]
```diff
@@ -88,7 +88,16 @@ public async Task<ActionResult<NOrders>> PostOrders(Orders order)
                     NOrderId = nOrder.Id,
                     SideId = sideId
                 };
-                await _orederRepo.Add(os);
+                await _orderRepo.Add(os);
+            }
+            foreach (var PreMadePizzaId in order.PreMadePizazIds)
```
Comment (steamwings) [inline PizzaShop/Controllers/OrdersController.cs]: same thing

File: PizzaData/Models/Orders.cs (hunk 1 above)
Comment (rwideman94) [reply PizzaData/Models/Orders.cs]: Fixed

File: PizzaShop/Controllers/OrdersController.cs (hunk 2 above)
Comment (rwideman94) [reply PizzaShop/Controllers/OrdersController.cs]: Whoopsy

**Review by steamwings — APPROVED** [review]
Looks tasty

**Pull request merged**